libraries/AC_AttitudeControl/AC_PosControl.h

```diff
@@ -317,18 +316,16 @@ class AC_PosControl
             uint16_t reset_desired_vel_to_pos   : 1;    // 1 if we should reset the rate_to_accel_xy step
             uint16_t reset_accel_to_lean_xy     : 1;    // 1 if we should reset the accel to lean angle step
             uint16_t reset_rate_to_accel_z      : 1;    // 1 if we should reset the rate_to_accel_z step
-            uint16_t freeze_ff_z        : 1;    // 1 used to freeze velocity to accel feed forward for one iteration
-            uint16_t use_desvel_ff_z    : 1;    // 1 to use z-axis desired velocity as feed forward into velocity step
             uint16_t vehicle_horiz_vel_override : 1; // 1 if we should use _vehicle_horiz_vel as our velocity process variable for one timestep
     } _flags;
 
     // limit flags structure
     struct poscontrol_limit_flags {
-        uint8_t pos_up      : 1;    // 1 if we have hit the vertical position leash limit while going up
-        uint8_t pos_down    : 1;    // 1 if we have hit the vertical position leash limit while going down
-        uint8_t vel_up      : 1;    // 1 if we have hit the vertical velocity limit going up
-        uint8_t vel_down    : 1;    // 1 if we have hit the vertical velocity limit going down
-        uint8_t accel_xy    : 1;    // 1 if we have hit the horizontal accel limit
+        bool pos_up;    // true if we have hit the vertical position leash limit while going up
+        bool pos_down;  // true if we have hit the vertical position leash limit while going down
+        bool vel_up;    // true if we have hit the vertical velocity limit going up
+        bool vel_down;  // true if we have hit the vertical velocity limit going down
+        bool accel_xy;  // true if we have hit the horizontal accel limit
```

Comment:
After the S-Curve navigation and input shaping is all in and being used I will be able to clean this up properly.
It is hard to do it while it is all in flux because I have to keep stuff in there to support code that will be removed and can't add code until we have other code to use it.

So I am happy for the code above to be removed and I will add it again later if needed.